src/telemetry/index.ts

```diff
@@ -8,14 +8,18 @@ export class TelemetryService implements vscode.Disposable {
     "50598369-dd83-4f9a-9a65-ca1fa6f1785c",
   );
 
+  private instanceName = vscode.workspace
+    .getConfiguration("dbt")
+    .get<string>("altimateInstanceName");
+
```

Comment:
maybe just inline this that we always latest setting, otherwise this would need a restart to be updated.